nextflow/MaveDB/main.nf

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env nextflow
+
+/* 
+ * Nextflow pipeline to create MaveDB plugin data for VEP
+ */
+
+nextflow.enable.dsl=2
+
+// Default params
+params.help     = false
+params.mappings = null
+params.ensembl  = "${ENSEMBL_ROOT_DIR}"
+params.output   = "output/MaveDB_variants.tsv.gz"
+
+params.licences = "CC0" // Open-access only
+params.round    = 4
+
+// Print usage
+if (params.help) {
+  log.info """
+  Create MaveDB data for VEP
+  --------------------------
+
+  Usage:
+    ./main.nf -profile lsf -resume \
+              --mappings [MaveDB_mappings_folder]
+
+  Mandatory arguments:
+    --mappings  Path to directory containing MaveDB mapping files in JSON format
+
+  Optional arguments:
+    --ensembl   Path to Ensembl root directory (default: ${ENSEMBL_ROOT_DIR})
+    --output    Path to output file (default: 'output/MaveDB_variants.tsv.gz')
+    --licences  Comma-separated list of accepted licences (default: 'CC0')
+    --round     Decimal places to round floats in MaveDB data (default: 4)
+
+  Note: the mappings must be named with the MaveDB accession, such as
+        '00000001-a-1.json'.
+  """
+  exit 1
+}
+
+// Module imports
+include { get_files_by_licence;
```

Comment:
can we have a more meaningful name instead of `licenses`? how about `fetch` as the main function is to fetch files?